include/mbgl/util/compression.hpp

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include <string>
+
+namespace mbgl {
+    namespace util {
```

Comment:
Nit: the convention inside mbgl is to avoid extra indentation due to namespaces.